resources/conf/wdio-bstack-flutterAndroid.conf.js

```diff
@@ -0,0 +1,48 @@
+var defaults = require("./wdio.conf.js");
+var _ = require("lodash");
+
+
+var overrides = {
+  user: process.env.BROWSERSTACK_USERNAME,
+  key: process.env.BROWSERSTACK_ACCESS_KEY,
+  specs: ["./src/test/flutterAndroid.spec.js"],
+  host: "hub.browserstack.com",
+  capabilities: [
+    {
+      maxInstances: 1,
+      "browserstack.debug": true,
+      automationName: "Flutter",
+      os_version: "11.0",
+      realMobile: 'true',
+      device: "Samsung Galaxy S21",
+      app: "bs://<app_id_camera_injection>", //pass the app_url for camera_injection apk placed under /resources/app directory
```

Comment:
instead of `//pass the app_url for camera_injection apk placed under /resources/app directory`

use
`//pass the app_url or custom id of the uploaded app on BrowserStack. 
Note: For camera_injection, sample apk for app upload is placed under /resources/app directory`